lib/task/propel/propelBuildNestedSetTask.class.php

```diff
@@ -35,6 +35,8 @@ class propelBuildNestedSetTask extends sfBaseTask
      */
     public function execute($arguments = [], $options = [])
     {
+        sfContext::createInstance($this->configuration);
+
```

Comment:
After extending from arBaseTask, we could replace the first four lines from this method with a call to the parent method. Check other tasks doing the same, like:

https://github.com/artefactual/atom/blob/7d52ca72de98df50b64f86bcd9f2b7a0fc805113/lib/task/tools/updatePublicationStatusTask.class.php#L53

Reply:
@anvit, I think you could also remove the two lines for the connection creation below:

https://github.com/artefactual/atom/blob/f4d2c1c502e88926c7124dfb91468e90446986fb/lib/task/propel/propelBuildNestedSetTask.class.php#L40-L41

Reply:
@jraddaoui I'd done that initially, but then noticed that these weren't actually being set up in arBaseTask's execute method, and also removing it made our `csv_import.cy.js` cypress test which checks for sibling order being preserved on import fail. 
Is there something I might be missing?

Reply:
I see, we are using transactions in this task which requires to create the connection like that instead of using the default one. Good thing you tested that out, thanks!